src/main/java/jenkins/plugins/slack/webhook/ListProjectsCommand.java

```diff
@@ -41,11 +33,17 @@ public SlackTextMessage execute(String... args) {
 
         String response = "*Projects:*\n";
 
+        Jenkins j = Jenkins.getInstance();
+        if (j == null) {
+            throw new IllegalStateException();
+        }
+
         List<AbstractProject> jobs =
-            Jenkins.getInstance().getAllItems(AbstractProject.class);
+            j.getAllItems(AbstractProject.class);
 
         SecurityContextHolder.setContext(ctx);
 
+        StringBuffer buf = new StringBuffer();
```

Comment:
If you operate in a single thread, use `StringBuilder`. It is *MUCH* more efficient, because it has no concurrency logic overheads